openid-4-verifiable-presentations-1_0.md

```diff
@@ -1583,21 +1625,33 @@ Note: If the Verifier's Response URI did not return a `redirect_uri` in step (6)
 
 # Security Considerations {#security_considerations}
 
-## Preventing Replay of the VP Token {#preventing-replay} 
+## Preventing Replay of Verifiable Presentations {#preventing-replay} 
+
+An attacker could try to inject Presentations obtained from (for example) a previous Authorization Response into another Authorization Response, thus impersonating the End-User that originally presented the respective Verifiable Presentation. Holder Binding aims to prevent such attacks.
+
+### Presentations without Holder Binding Proofs
 
-An attacker could try to inject a VP Token (or an individual Verifiable Presentation), that was obtained from a previous Authorization Response, into another Authorization Response thus impersonating the End-User that originally presented that VP Token or the respective Verifiable Presentation.
+By definition, Presentations without Holder Binding (see (#nkb-credentials)) do
+not prove that the Holder possesses the private Holder Binding key belonging to
+the Credential. A Verifier that consumes Presentations without Holder Binding
+accepts the risk that the Holder may have obtained the Credential from a third
+party (e.g., by playing the role of a Verifier) and that the Holder may not be
```

Comment:
I suggest to add considerations how the binding could be achieved, like claims binding to other credentials.

Reply:
From my experience with similar discussions in SD-JWT, this is a bit of a rathole and there are a lot of opinions on this topic :-) I suggest to file an issue and add text later.

Reply:
https://github.com/openid/OpenID4VP/issues/533